src/TropicalGeometry/positive_variety.jl

```diff
@@ -0,0 +1,73 @@
+@doc raw"""
+    positive_tropical_variety(I::MPolyIdeal,nu::TropicalSemiringMap)
+
+Return the positive tropical variety of `I` as a `PolyhedralComplex` as per the definition in [SW05](@cite).  Assumes that `I` is generated either by binomials or by linear polynomials and that `I` is defined either over
```

Comment:
```suggestion
Return the positive tropical variety of `I` as a `PolyhedralComplex` as per the definition in [SW05](@cite).

Assumes that `I` is generated either by binomials or by linear polynomials and that `I` is defined either over
```
Break an overly long line. And good docstring style adds a paragraph break after the first sentence